pkg/Microsoft.Private.PackageBaseline/stable.packages.props

```diff
@@ -1479,5 +1479,9 @@
     <StablePackage Include="System.Security.Cryptography.X509Certificates">
       <Version>4.1.2</Version>
     </StablePackage>
+
```

Comment:
Would you mind adding the comment of what servicing wave this is? So that later is easier to read in what framework version we shipped that stable package?

Reply:
Not really part of a wave so I left it blank but I did put it in the order it was shipped.

Reply:
Yup. That is fine, didn't know until I talked to you offline :)